prez-components/src/stories/PrezUINode.stories.ts

```diff
@@ -0,0 +1,49 @@
+import type { Meta, StoryObj } from "@storybook/vue3";
+import { vueRouter } from "storybook-vue3-router";
+import { node, nodeLink, nodePredicate } from "../util/story-data/PrezUINode";
+import PrezUINode from "../components/PrezUINode.vue";
+
+const meta = {
+    title: "PrezUINode",
+    component: PrezUINode,
+    tags: ["autodocs"],
+    argTypes: {
+        iri: { description: "The IRI of the node", type: "string" },
+        label: { description: "The node's label" },
+        description: { description: "The node's description" },
+        provenance: { description: "The node's provenance - used for a custom explanation of how this node/predicate is used in Prez" },
+        curie: { description: "The node's curie, a shorthand notation using the namespace's prefix", type: "string" },
+        links: { description: "An array of Prez links" },
+        rdfTypes: { description: "An array of types" },
+        showType: { description: "Toggles showing the type badges", type: "boolean" },
+        showProv: { description: "Toggles showing the provenance tooltip", type: "boolean" },
+        termType: { table: { disable: true } }
+    },
+} satisfies Meta<typeof PrezUINode>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
```

Comment:
1. Don't know it'll be easy to follow all the test cases are being covered with the separate story data definition vs inline.
2. Previously the PrezUINode had one property for the term, and then you can have additional UI properties for the UI component. This separated the data property (term) from the UI properties. Aggregating the properties into the object blurs the distinction between the data object property and the UI properties.

Reply:
1. Are we reading the story data in story files or in Storybook?
2. Separating objects in prop types seems like an unnecessary separation to me

Reply:
1. Might be easier to walk through the scenario with number of anticipated test cases.
2. I see the separation of concerns as necessary as it clearly allows you to access or share the data object that the UI component is using.